openapi-cli/src/main/java/io/ballerina/openapi/generators/schema/TypeGeneratorUtils.java

```diff
@@ -155,17 +161,36 @@ public static TypeDescriptorNode getNullableType(Schema schema, TypeDescriptorNo
             MetadataNode metadataNode = createMetadataNode(documentationNode, createEmptyNodeList());
             if (required != null) {
                 if (!required.contains(field.getKey().trim())) {
-                    recordFieldNode = NodeFactory.createRecordFieldNode(metadataNode, null,
-                            fieldTypeName, fieldName, createToken(QUESTION_MARK_TOKEN), createToken(SEMICOLON_TOKEN));
+                    if (field.getValue().getDefault() != null) {
+                        Token defaultValue;
+                        if ((field.getValue().getDefault()).getClass() == String.class) {
```

Comment:
could you please let me know if we have an empty string as default value ` ex: url:" "`, what would be our output, are we going to add an empty string there? 🙂

Reply:
```yaml
components:
  schemas:
    Pet:
      required:
        - id
        - name
      properties:
        id:
          type: integer
          format: int64
        name:
          type: string
        tag:
          type: string
          default: ""
        tagNumber:
          type: integer
          default: 10
        type:
          type: string
```
**Generated code** 
```bal
public type Pet record {
    int id;
    string name;
    string tag = "";
    int tagNumber = 10;
    string 'type?;
};
```

Reply:
Yeah, it is fine for me to pick an empty string there since the user adds it intentionally. :) it would be great if we can have ideas from @aneeshafedo @indikasampath2000  🙂

Reply:
+1